tests/lib/rules/no-array-prototype-extensions.js

```diff
@@ -36,9 +36,9 @@ ruleTester.run('no-array-prototype-extensions', rule, {
     /** Optional chaining */
     'arr?.notfirstObject?.foo',
     'arr?.filter?.()',
-    /** Replace is part of string native prototypes */
-    "'something'.replace()",
-    'something.replace()',
+    /** String.prototype.replace() */
+    "'something'.replace(regexp, 'substring')",
+    "something.replace(regexp, 'substring')",
```

Comment:
Can we add valid test cases for:
* `replace()`
* `replace(foo)`
* `replace(foo, bar, baz)`

Reply:
Yes added!